instance_config.yaml

```diff
@@ -138,7 +138,6 @@ val_profile_dict:
     override:
         - desktop_mp4
         - mobile_low
-        - mobile_high
```

Comment:
Don't we need this anymore ? We should discuss the final encodes that we'll provide, we'll update VAL. Also, this should be communicated to mobile as well.

Reply:
@muzaffaryousaf It was never used in mobile, and I think HLS is going to supplant it, but it can stay in VAL for now, there are still existing legacy mobile_high encodes. If we delete them all from VAL, we can delete it there.

Reply:
It's safe.